### PR TITLE
Merge upstream HEAD(e68103e03, 2024-05-14) Remove V2 tables

### PR DESCRIPTION
**Pull request by milo-hyben**
Conflicts reported:

```
both modified:   .gitignore
deleted by them: hail/build.gradle
both modified:   hail/python/hailtop/httpx.py
```

	
Resolved as per comments.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -50,3 +50,4 @@ hail/python/hail/backend/extra_classpath
 hail/python/hail/backend/hail.jar
 hail/install-editable
 _/
+.helix
```
Comment (milo-hyben) [inline .gitignore:53]: Incoming, added .helix:

```
+<<<<<<< HEAD
+_/
+||||||| b7bde56d5
+=======
 .helix
+>>>>>>> e68103e03
```

File: hail/build.gradle [hunk 2]
```diff
@@ -1,364 +0,0 @@
-buildscript {
```
Comment (milo-hyben) [inline hail/build.gradle:1]: Incoming:
file removed

File: hail/python/hailtop/httpx.py [hunk 3]
```diff
@@ -104,27 +104,45 @@ def __init__(self,
 
         if timeout is None:
```
Comment (milo-hyben) [inline hail/python/hailtop/httpx.py:105]: Merged both:
```
+<<<<<<< HEAD
+        if timeout is None:
+            timeout = aiohttp.ClientTimeout(total=20)
+        if isinstance(timeout, (float, int)):
+            timeout = aiohttp.ClientTimeout(total=timeout)
+||||||| b7bde56d5
+        if timeout is None:
+            timeout = aiohttp.ClientTimeout(total=5)
+        if isinstance(timeout, (float, int)):
+            timeout = aiohttp.ClientTimeout(total=timeout)
+=======
         configuration_of_timeout = configuration_of(ConfigVariable.HTTP_TIMEOUT_IN_SECONDS, timeout, 5)
         del timeout
 
@@ -110,6 +121,7 @@ class ClientSession:
         if isinstance(configuration_of_timeout, (float, int)):
             configuration_of_timeout = aiohttp.ClientTimeout(total=configuration_of_timeout)
         assert isinstance(configuration_of_timeout, aiohttp.ClientTimeout)
+>>>>>>> e68103e03
```

File: hail/build.gradle (hunk 2 above)
Comment (jmarshall) [reply hail/build.gradle:1]: Yep, we just added a gradle plugin called `bloop`, which speeds up scala builds. Should be fine to just drop it.

File: hail/python/hailtop/httpx.py (hunk 3 above)
Comment (jmarshall) [reply hail/python/hailtop/httpx.py:105]: Remember that our local change is just to change the default from 5 to 20.

They've refactored this to introduce this `configuration_of_timeout` stuff. So I would use their version, drop the existing four lines, and change the default on the new first line (i.e., the `configuration_of_timeout = configuration_of(…)` line) from 5 to 20.

**Review by jmarshall — APPROVED** [review]

File: hail/python/hailtop/httpx.py (hunk 3 above)
Comment (milo-hyben) [reply hail/python/hailtop/httpx.py]: Yes, silly me :-), makes sense.

**Pull request merged**